### PR TITLE
Bugfix/see all result doesn't have breakpoints

### PR DESCRIPTION
**Pull request by Gusvarov**
Before
image
After
image

### CODE REVIEW

File: src/app/component/layout/components/search-all-results/search-all-results.component.scss [hunk 1]
```diff
@@ -136,3 +136,39 @@ $main-font-color: #494A49;
     }
   }
 }
+
```
Comment (mr-pavlenko) [inline src/app/component/layout/components/search-all-results/search-all-results.component.scss:139]: Why did you use rem? Mix up px rem and % is bad practice.

Comment (mplesha) [reply src/app/component/layout/components/search-all-results/search-all-results.component.scss:139]: agree

**Comment by mplesha** [comment]
agree with comment - it's bad to mix

File: src/app/component/layout/components/search-all-results/search-all-results.component.scss (hunk 1 above)
Comment (Gusvarov) [reply src/app/component/layout/components/search-all-results/search-all-results.component.scss:139]: fixed

**Pull request merged**